lib/src/models/subreddit.dart

```diff
@@ -747,7 +696,7 @@ class SubredditStream {
   /// will initially be returned. If [pauseAfter] is provided, the stream will
   /// return null after [pauseAfter] iterations.
   Stream<Submission> submissions({int pauseAfter}) =>
-      streamGenerator(_subreddit.submissions, pauseAfter: pauseAfter);
+      streamGenerator(_subreddit.newest, pauseAfter: pauseAfter);
```

Comment:
I see the following method in `base.dart`,

```dart
  /// Returns a [Stream] of the newest comments and submissions.
  Stream<UserContent> newest({Map params}) => _buildGenerator(params, 'new');
```

However you seem to have two different signatures for calling the method:
1.) `subreddit.newest();`
2.) `_subreddit.newest`

This question is mostly out of curiosity, but why is this?